tests/functional/stax/test_ui.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from ragger.navigator import NavIns, NavInsID
+from typing import Iterable, Union
+
+from .navigator import CustomNavInsID
+
+
+def format_instructions(instructions: Iterable[Union[NavIns, CustomNavInsID]]) -> Iterable[NavIns]:
+    return [NavIns(instruction) if isinstance(instruction, CustomNavInsID) else instruction
+            for instruction in instructions]
+
+
+@pytest.mark.use_on_firmware("stax")
+def test_immediate_quit(navigator):
+    instruction = format_instructions([CustomNavInsID.HOME_TO_QUIT])
+    navigator.navigate(instruction,
+                       screen_change_before_first_instruction=False,
+                       screen_change_after_last_instruction=False)
+
+
+@pytest.mark.use_on_firmware("stax")
+def test_settings_screens(navigator):
+    instructions = format_instructions([
+        CustomNavInsID.HOME_TO_SETTINGS,
+        CustomNavInsID.SETTINGS_TO_HOME,
+        CustomNavInsID.HOME_TO_QUIT
+    ])
+    navigator.navigate(instructions,
+                       screen_change_before_first_instruction=False,
+                       screen_change_after_last_instruction=False)
+
+
+@pytest.mark.use_on_firmware("stax")
+def test_delete_one_password(navigator, functional_test_directory):
+    instructions = format_instructions([
+        CustomNavInsID.HOME_TO_MENU,
+        # ensure the password list is filled with populated passwords
+        CustomNavInsID.MENU_TO_DELETE,
+        # choose the password to delete then confirm
+        NavIns(CustomNavInsID.LIST_CHOOSE, (2, )),
+        CustomNavInsID.CONFIRM_YES,
+        NavIns(NavInsID.WAIT, (2, )),
+        # check the password has been removed from the list
+        CustomNavInsID.MENU_TO_DISPLAY,
+        CustomNavInsID.LIST_TO_MENU
+    ])
+    navigator.navigate_and_compare(functional_test_directory,
+                                   "delete_one_password",
+                                   instructions,
+                                   screen_change_before_first_instruction=False,
+                                   screen_change_after_last_instruction=False)
```

Comment:
could be true here as you don't quit the app? Probably same on different scenario below.

Reply:
Indeed, however I don't really care of checking this last screen. I'll add it once, and remove the last instruction on the other (and leave ` screen_change_after_last_instruction` to its default value).